apps/dokploy/templates/vaultwarden/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+services:
+  vaultwarden:
+    image: vaultwarden/server:1.32.3
+    restart: always
+    environment:
+      DOMAIN: ${DOMAIN}
+      SIGNUPS_ALLOWED: ${SIGNUPS_ALLOWED}
+    volumes:
+      - vaultwarden:/data
+    ports:
+      - 11001:80
```

Comment:
I tested this template and it doesn't work, also I tried to remove the expose port, is there any way you are exposing this port?

Reply:
![image](https://github.com/user-attachments/assets/bca474ce-62a2-4490-930b-5a3d723f1f09)
So strange, in internal network works fine.